tests/ProportionalControllerTest.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file ProportionalControllerTest.cpp
+ * @author Giulio Romualdi
+ * @copyright Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2020
+ */
+
+// Catch2
+#include <catch2/catch.hpp>
+
+#include <LieGroupControllers/ProportionalController.h>
+using namespace LieGroupControllers;
+
+TEST_CASE("Proportional Controller [SO(3)]")
+{
+    manif::SO3d desiredState, state;
+    desiredState.setRandom();
+    state.setRandom();
+
+    constexpr double dT = 0.01;
+    constexpr double kp = 10;
+    constexpr std::size_t numberOfIteration = 1e3;
+
+    ProportionalControllerSO3 controller;
+    controller.setGain(kp);
+    controller.setDesiredState(desiredState);
+
+    auto feedForward = Eigen::Vector3d::Zero();
+    controller.setFeedForward(feedForward);
+
+    for (std::size_t i = 0; i < numberOfIteration; i++)
+    {
+        controller.setState(state);
+        controller.computeControlLaw();
+        auto controlOutput = controller.getControl();
+        decltype(controlOutput) controlOutputDT = controlOutput.coeffs() * dT;
+        state = controlOutputDT + state;
+    }
+
+    // check the error
+    auto error = state.compose(desiredState.inverse()).log();
```

Comment:
Is this the same as `lminus` ?  What would be the difference if you called `desiredState.compose(state.inverse()).log();`?

Reply:
This is indeed the same as `lminus` (see [lminus implementation](https://github.com/artivis/manif/blob/devel/include/manif/impl/lie_group_base.h#L462)).

Reply:
It should be exactly the same